conformance/ipv6.go

```diff
@@ -0,0 +1,196 @@
+					pingPod(ip, testNode, ipv6NetworkName)
+				}
+			})
+		})
+	})
+})
+
+func createSriovNetwork() {
+	sriovNetwork := &sriovv1.SriovNetwork{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ipv6NetworkName,
+			Namespace: operatorNamespace,
+		},
+		Spec: sriovv1.SriovNetworkSpec{
+			ResourceName:     resourceName,
+			IPAM:             `{"type": "host-local","ranges": [[{"subnet": "3ffe:ffff:0:01ff::/64"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`,
+			NetworkNamespace: testNamespace,
+		}}
+	err := clients.Create(context.Background(), sriovNetwork)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+
+func createSriovPolicy(node string, sriovDevice string, testNode string) {
+	nodePolicy := &sriovv1.SriovNetworkNodePolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-policy-",
+			Namespace:    operatorNamespace,
+		},
+		Spec: sriovv1.SriovNetworkNodePolicySpec{
+			NodeSelector: map[string]string{
+				"kubernetes.io/hostname": testNode,
+			},
+			NumVfs:       numVfs,
+			ResourceName: resourceName,
+			Priority:     99,
+			NicSelector: sriovv1.SriovNetworkNicSelector{
+				PfNames: []string{sriovDevice},
+			},
+			DeviceType: "netdevice",
+		},
+	}
+
+	err := clients.Create(context.Background(), nodePolicy)
+	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func() bool {
+		stable, err := cluster.SriovStable(operatorNamespace, clients)
+		Expect(err).ToNot(HaveOccurred())
+		return stable
+	}, 5*time.Minute, 1*time.Second).Should(Equal(true))
+
+	Eventually(func() int64 {
+		testedNode, err := clients.Nodes().Get(testNode, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		resNum, _ := testedNode.Status.Capacity["openshift.io/" + resourceName]
+		capacity, _ := resNum.AsInt64()
+		return capacity
+	}, 3*time.Minute, time.Second).Should(Equal(int64(numVfs)))	
+}
+
+func getSriovNicIps(pod *k8sv1.Pod) []string {
+	var nets []Network
+	err := json.Unmarshal([]byte(pod.ObjectMeta.Annotations["k8s.v1.cni.cncf.io/networks-status"]), &nets)
+	Expect(err).ToNot(HaveOccurred())
+	for _, net := range nets {
+		if net.Interface != sriovInterfaceName {
+			continue
+		}
+		return net.Ips
+	}
+	return nil
+}
+
+func createTestPod(command []string, args []string, node string, sriovNetworkAttachment string) *k8sv1.Pod {
+	podDefinition := testPodDefintion([]string{"/bin/bash", "-c", "--"},
+		[]string{"while true; do sleep 300000; done;"}, node, sriovNetworkAttachment)
+	createdPod, err := clients.Pods(testNamespace).Create(podDefinition)
+
+	Eventually(func() k8sv1.PodPhase {
+		runningPod, err := clients.Pods(testNamespace).Get(createdPod.Name, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		return runningPod.Status.Phase
+	}, 3*time.Minute, 1*time.Second).Should(Equal(k8sv1.PodRunning))
+	pod, err := clients.Pods(testNamespace).Get(createdPod.Name, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	return pod
+}
+
+func pingPod(ip string, nodeSelector string, sriovNetworkAttachment string) {
```

Comment:
can you move this function into the `utils/pod` package?

Reply:
The function is mainly about Expect's/Eventually's, so keeping some of it here.
Moved the test pod definitinion functions to utils